pkg/kapp/cmd/app/deploy_flags.go

```diff
@@ -5,6 +5,7 @@ package app
 
 import (
 	"fmt"
+
 	"github.com/spf13/cobra"
 
```

Comment:
nit: remove new line

Reply:
hmm, I think my IDE put that in there, will remove 👍

Reply:
everytime I save in VSCode it brings the line back 🙄 .

Reply:
Actually I was talking about the newline after `"github.com/spf13/cobra"`, which I think has been there for some time now 😅 . We usually group the imports into 2 sections (separated by a new line), where first section consists of the standard libraries provided by Go (`fmt` in this case) and rest all go into the second section. 
So the new line you had added is perfectly fine, we just need to remove the other new line.

I shouldn't have mentioned it as `nit` I guess, my bad 😅

Reply:
ahhhhh, gotcha. I was confused because I thought I introduced this, whereas you were asking to help fixup the existing thing. Gotcha! I fixed up it, `go fmt` rearranged the second set of imports so the named one was first also.